controllers/csi/provisioner/reconciler.go

```diff
@@ -139,7 +144,8 @@ func (r *OneAgentProvisioner) Reconcile(ctx context.Context, request reconcile.R
 			err = r.db.UpdateTenant(tenant)
 		}
 		if err != nil {
-			return reconcile.Result{}, err
+			rlog.Error(err, "error when updating tenant")
```

Comment:
not needed, error is logged anyway after returning

Reply:
done